### ISSUE TITLE
1.2.1 language to be used in media-alternatives for sign-language videos

### ISSUE THREAD

**Issue by ebalink** [w3c/wcag#1910]
The alternative for time-based media needs to be provided in the same language as the original content.

(Most) spoken languages have an orthographic system to define how the spoken language should be spelled to a written form. Sign languages do not. Nor do they have official versions of how they should be spoken. In the definitions it's correctly stated that [sign languages](https://www.w3.org/TR/WCAG21/#dfn-sign-language-interpretation) are separate from their "corresponding" spoken languages. This means that transition from a spoken language to a sign language is always an interpretation. 

So if there's a sign-language video (video-only content), it needs to meet[ the criterion 1.2.1](https://www.w3.org/TR/WCAG21/#audio-only-and-video-only-prerecorded) by providing an alternative for time-based media or an audio track. Be definition this should be done with same language as the original one but for sign language content it's not possible. It would always require translation to some spoken language (and it's not clear to which one). 
In WCAG, it should be defined whether this frees the sign-language videos from providing transcripts or should the requirement be met by translating the content to a suitable spelled language. 

If a video has no audio track and contains only signs it's not accessible to users with limited vision. So this is more than just a theoretical question. The requirement is somewhat a counterpart for the transcript requirement that applies also for podcasts of the visually impaired communities.

**Comment by detlevhfischer** [comment]
The SC 1.2.1 says:
> For prerecorded audio-only and prerecorded video-only media, the following are true, except when the audio or video is a media alternative for text and is clearly labeled as such: (...)

I guess in most cases, sign language videos are going to be considered a media alternative for text (even though they may summarise, synthesise, transform the text content of a site), so with that view, there would be no requirement for a media alternative.

**Comment by patrickhlauke** [comment]
at a very high level - and i'm far from an expert here, so forgive me if i get this wrong - there are a lot of different national/regional variations for sign language (https://en.wikipedia.org/wiki/List_of_sign_languages#Contemporary_deaf_sign_languages) which do map to specific written languages. so my assumption would be if a video only contains British Sign Language, that its transcript would need to be in `en-UK`. where there's no clear mapping, it may get trickier, and i'd say you probably will want to be guided by what the natural language of the rest of the page/site is.

**Comment by ebalink** [comment]
@detlevhfischer , yes. For most cases but my understanding is that for sign language communities there are web sites that contain mainly sign-language videos without any text counterpart. 

In the big picture it’s clear that WCAG improves the web accessibility for hearing impaired users, but as WCAG is references in multiple legislations, it would be good for the guidelines to be clear that the requirements are ment to apply to all web sites – including the ones of persons with disabilities.
